cppseminar/presentation/Services/SubmissionService.cs

```diff
@@ -58,6 +58,45 @@ public async Task<IList<Submission>> GetUserSubmissionsAsync(string userEmail)
             }
         }
 
+        public async Task<List<long>> GetCounts(string userEmail)
```

Comment:
GetCounts of what? This name is not in line with previous function in this file and is very confusing to me.